### PR TITLE
Added character <a> for <starwar>

### LINKED ISSUE #417: Add character: <a> to font: <starwar>
*Character* : a (lowercase)
*Font* : starwar

### PR DESCRIPTION
**Pull request by s-xiae**
Added Character \<a> to font \<starwar>

Lowercase a isn't in the code, so I added the code for it (it's the same as uppercase A)
Even though the ascii art for both A's are the same, the input would be different for both, so this way the computer would read both no problem.

closes #417

### CODE REVIEW

**Comment by AnshumanMahato** [comment]
Grate. That's exactly what had to be done. 🫡

By the way, for further PRs, with for any collaborator to assign the issue to you before you make a PR. This is to ensure that duplicates don't happen. Also, don't close issues by yourself. Just link it to the PR and leave the rest.

Anyways, thanks for contributing. 😊

File: Fonts/starwar/starwar.h [hunk 1]
```diff
@@ -92,16 +92,17 @@ class Starwar : public Fonts
         return character;
     }
 
-    /*
-        char **G()
-        {
-            char **character = getCharGrid(ROWS,COLS);
+    char **G()
+    {
+        char **character = getCharGrid(ROWS,COLS);
```
Comment (AnshumanMahato) [inline Fonts/starwar/starwar.h:97]: pass the exact no. of rows and columns that you used to make your character.

File: Fonts/starwar/starwar.h [hunk 2]
```diff
@@ -352,16 +353,18 @@ class Starwar : public Fonts
...
 
-    /*
-        char **a()
-        {
-            char **character = getCharGrid(ROWS,COLS);
 
-            //Enter the character grid in ROWS X COLS
+    char **a()
+    {
+        char **character = getCharGrid(ROWS,COLS);
```
Comment (AnshumanMahato) [inline Fonts/starwar/starwar.h:359]: same here as well

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]

File: Fonts/starwar/starwar.h (hunk 1 above)
Comment (s-xiae) [reply Fonts/starwar/starwar.h:97]: added the values

File: Fonts/starwar/starwar.h (hunk 2 above)
Comment (s-xiae) [reply Fonts/starwar/starwar.h]: added the values here too.

**Review by AnshumanMahato — APPROVED** [review]

**Pull request merged**